jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfig.java

```diff
@@ -65,18 +60,8 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.StringReader;
-import java.sql.Connection;
-import java.sql.DatabaseMetaData;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Properties;
-import java.util.Set;
+import java.sql.*;
+import java.util.*;
```

Comment:
just one thing, quickly. I am not again refactoring at all, but please do this in separate pull request, dedicated to refactoring and only refactoring. When opening a pull request for feature or bugfix, please do changes ONLY needed for that.
It is easier to read and approve, and easier to cherry-pick if needed later

Reply:
Yes, I see your point. Actually, this was an automatic clean from IntelliJ, so it was not on purpose. From my job, I am get used to automatic formatting, but in these projects we have defined rules, so every IDE from each developers apply the same formatting. Since we do not have rules here it totally makes sense to deactivate automatic formatting. I will remove these changes later during the day. Thanks for the hint! 😃

Reply:
Same thing for me, with Eclipse ;)
I end with a complete different workspace, dedicated for open source projects

Reply:
Thats another option yes :)

I tried to revert all irrelevant formatting changes. Let me know if there's anything else I can do, to make the review easier.